examples-gallery/plot_ball_rolling_on_spinning_disc.py

```diff
@@ -349,21 +349,19 @@ def obj_grad(free):
 )
 
 # %%
-# The initial guess should meet the configuration constrains. It will be
-# plotted below.
-i1b = np.zeros(num_nodes)
-i2 = np.linspace(initial_state_constraints[x],
-    final_state_constraints[x], num_nodes)
-i1a = i2 / par_map[r]
-i3 = np.linspace(initial_state_constraints[y],
-    final_state_constraints[y], num_nodes)
-i1 = -i3 / par_map[r]
-i4 = np.zeros(8*num_nodes)
-initial_guess = np.hstack((i1,i1a, i1b, i2, i3, i4, 0.01))
-
-fig1, ax1 = plt.subplots(14, 1, figsize=(7.25, 0.75*14), sharex=True,
-    layout='constrained')
-prob.plot_trajectories(initial_guess, ax1)
+# The initial guess should meet the configuration constrains. Those below were
+# chosen to get a better initial guess, now stored in
+# 'ball_rolling_on_spinning_disc_solution.npy'.
+
+# -i1b = np.zeros(num_nodes)
+# -i2 = np.linspace(initial_state_constraints[x],
+#    final_state_constraints[x], num_nodes)
+# -i1a = i2 / par_map[r]
+# -i3 = np.linspace(initial_state_constraints[y],
+#    final_state_constraints[y], num_nodes)
+# -i1 = -i3 / par_map[r]
+# -i4 = np.zeros(8*num_nodes)
+# -initial_guess = np.hstack((i1,i1a, i1b, i2, i3, i4, 0.01))
```

Comment:
Also, for examples it is best not to have commented code. You can create this initial guess and then overwrite it with the npy file just like you did in the other.